### ISSUE TITLE
Figure out what's wrong with Windows builds

### ISSUE THREAD

**Issue by tomusdrw** [paritytech/jsonrpc#362]
They keep failing, maybe we should get appveyor back? CC @ordian

**Comment by ordian** [comment]
I agree, that to travis for Windows was premature. We should switch back appveyor till https://travis-ci.community/t/windows-instances-hanging-before-install/250 is resolved.
